### PR TITLE
Added seqstat, set minlength for cutadapt to 1. 

### PR DESCRIPTION
**Pull request by rhpvorderman**
Also submitted issue upstream https://github.com/marcelm/cutadapt/pull/310

toolJars are now optional files as they should be.

### CODE REVIEW

File: biopet.wdl [hunk 1]
```diff
@@ -276,3 +276,32 @@ task ScatterRegions {
     }
 }
 
+task Seqstat {
+    String? preCommand
+    File? toolJar
+    File fastq
+    String outputFilePath
```
Comment (DavyCats) [inline biopet.wdl]: 'File' and 'Path' seems a bit... dubbelop :stuck_out_tongue:

File: biopet.wdl [hunk 2]
```diff
@@ -276,3 +276,32 @@ task ScatterRegions {
...
+    Float? memory
+    Float? memoryMultiplier
+    Int mem = ceil(select_first([memory, 4.0]))
+
+    String toolCommand = if defined(toolJar)
+    then "java -Xmx" + mem + "G -jar " +toolJar
```
Comment (DavyCats) [inline biopet.wdl]: missing space before `toolJar`

File: cutadapt.wdl [hunk 3]
```diff
@@ -33,7 +33,7 @@ task cutadapt {
     String? stripSuffix
     String? prefix
     String? suffix
-    Int? minimumLength
+    Int? minimumLength = 1  # Necessary to prevent creation of empty reads
```
Comment (DavyCats) [inline cutadapt.wdl]: Default could be higher as far as I'm concerned. Maybe 20 or 30?

Does this way of defining defaults work now?

Comment (ffinfo) [reply cutadapt.wdl]: does not matter a lot I think, small reads won't align anyway. Aslong the user can set this is needed,

Comment (rhpvorderman) [reply cutadapt.wdl]: Best to keep the information by default. This parameter is just set so empty reads are filtered. Empty reads sometimes make tools crash.

**Pull request merged**